hack/test-go.sh

```diff
@@ -26,24 +26,11 @@ find_test_dirs() {
     \) -name '*_test.go' -print0 | xargs -0n1 dirname | sort -u | xargs -n1 printf "${S2I_GO_PACKAGE}/%s\n"
 }
 
-# -covermode=atomic becomes default with -race in Go >=1.3
-if [ -z ${S2I_COVER+x} ]; then
-  S2I_COVER=""
-fi
-
-OUTPUT_COVERAGE=${OUTPUT_COVERAGE:-""}
-
-if [ -n "${OUTPUT_COVERAGE}" ]; then
-  if [ -z ${S2I_RACE+x} ]; then
-    S2I_RACE="-race"
-  fi
-  if [ -z "${S2I_COVER}" ]; then
-    S2I_COVER="-cover -covermode=atomic"
-  fi
-fi
-
 if [ -z ${S2I_RACE+x} ]; then
-  S2I_RACE=""
+  S2I_RACE="-race"
+fi
+if [ -z ${S2I_COVER+x} ]; then
```

Comment:
Even more straightforward, no if:

```bash
S2I_COVER="${S2I_COVER-"-covermode=atomic"}"
```

Or we can just use this to set the value if-not-unset on first usage:

```bash
# ...

# parameter substitution, no assignment:
go test ... "${S2I_COVER-"-covermode=atomic"}"
# ^^^ I think we probably just need this one above, all other occurrences of S2I_COVER may disappear.

# substitution and assignment:
go test ... "${S2I_COVER="-covermode=atomic"}"
```

(Those expansions work with `nounset`)